aria_data_loaders/aria_data_utils/aria_streamer.py

```diff
@@ -137,22 +143,6 @@ def plot_rgb_and_trajectory(
         plt_ax.plot(traj_data[:, 0], traj_data[:, 1], traj_data[:, 2])
         plt.show(block=block)
 
-    def _rotate_img(self, img: np.ndarray, num_of_rotation: int = 3) -> np.ndarray:
-        """
-        Rotate image in multiples of 90d degrees
-
-        Args:
-            img (np.ndarray): Image to be rotated
-            k (int, optional): Number of times to rotate by 90 degrees. Defaults to 3.
-
-        Returns:
-            np.ndarray: Rotated image
-        """
-        img = np.ascontiguousarray(
-            np.rot90(img, k=num_of_rotation)
-        )  # GOD KNOW WHY THIS IS NEEDED -> https://github.com/clovaai/CRAFT-pytorch/issues/84#issuecomment-574683857
-        return img
-
     def _display(self, img: np.ndarray, stream_name: str, wait: int = 1):
```

Comment:
Could we remove _create_display_window() and _display() functions based on the conversation on slack?

Reply:
Thanks for pointing this out, addressed.